recipes/gtlab-logging/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools import files
+from conan.tools.build import check_min_cppstd
+from conan.errors import ConanInvalidConfiguration
+from conan import ConanFile
+import os
+
+required_conan_version = ">=1.59.0"
+
+class GTLabLoggingConan(ConanFile):
+    name = "gtlab-logging"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dlr-gtlab/gt-logging"
+    topics = ("logging", "qt")
+    description = "Simple logging interface with qt support"
+
+    settings = "os", "arch", "compiler", "build_type"
+    
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_qt": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_qt": False
+    }
+
+    def requirements(self):
+        if self.options.with_qt:
+            self.requires("qt/[>=5.0.0]")
```

Comment:
Version ranges are only allowed for a small subset of packages and Qt is not one of them, unfortunately.

Reply:
Qt is used only in the header files, i. e. the dependency is defined only by the package consumers. Hence, I disagree to fix a certain qt version. This package has no binary dependency to Qt! Thus, it can be used for any qt version that the consumer is using.

Reply:
Ok, makes sense. I would keep it as is. Thank you for the clarification!

Reply:
> Ok, makes sense. I would keep it as is. Thank you for the clarification!

Thank you!